examples/asset_licences.md

```diff
@@ -0,0 +1,9 @@
+The "Tuffy" font is public domain, was made by Thatcher Ulrich, and was accessed from here: http://www.publicdomainfiles.com/show_file.php?id=13486218041168
+The loop doodle_pop.ogg is public domain and was made by MrZeusTheCoder.
+opengl/resources/background.png is public domain, was made by Nidhoggn, and was accessed from here: https://opengameart.org/content/backgrounds-3
+shader/resources/background.jpg is public domain, was taken by Arcana Dea, and was accessed from here: https://www.publicdomainpictures.net/en/view-image.php?image=10979&picture=monarch-butterfly
+shader/resources/devices.png is public domain, was made by Kenny, and was accessed from here: https://www.kenney.nl/assets/game-icons
+sound/resources/ding.flac is public domain, was made by Kenny, and was accessed from here: https://www.kenney.nl/assets/interface-sounds
+sound/resources/killdeer.wav is public domain, was recorded by the US National Park Services, and was accessed from here: https://www.nps.gov/subjects/sound/sounds-killdeer.htm
+tennis/resources/ball.wav is public domain and was made by MrZeusTheCoder.
+win32/resources/image1.jpg & image2.jpg are public domain and have assets from here: https://www.kenney.nl/assets/toon-characters-1
```

Comment:
Since this file is a markdown file, could you maybe add some styling to it?
For example make this an actual list, put the file path into inline code blocks, and putting the link behind the `here` text.
Or alternatively, you could make a table with columns file, license, author, url

Reply:
Refer to my other answer on whether this was even going to be a thing.